setup.py

```diff
@@ -57,36 +57,27 @@ def detect_model_server_version():
     return __version__
 
 
-def add_requirements_to_2_x(pkg_name):
-    import sys
-    if sys.version_info[0] < 3:
-        requirements.append(pkg_name)
-
-
 def select_mxnet():
```

Comment:
Wondering if we should move "mxnet" pacakge to "extras_requires" .. This might be a better way to install mxnet at runtime. 
```
...
setup()
...
        extras_require={
            'mx_cpu': ['mxnet-mkl==1.2.0'],
            'mx_gpu': ['mxnet-cu90mkl>=1.2'],
        },
...
```
This way, when we install we could ask customers to install as
```
# MMS on CPU
$ pip install -U mxnet-model-server[mx_cpu]

# MMS on GPU
$ pip install -U mxnet-model-server[mx_gpu]
```

Can't think of a better way to install mxnet version based on CPU . vs GPU.